### PR TITLE
binlog entity: add meta in entity

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
for pump and drainer, can make code simple.

### CODE REVIEW

File: proto/binlog/pump.proto [hunk 1]
```diff
@@ -54,6 +54,9 @@ message Pos {
 
     // The binlog offset in a file.
     int64 offset = 2;
+
+     // The binlog's ts, start ts for prewrite binlog, and commit ts for commit binlog.
+    int64 ts = 3;
```
Comment (GregoryIan) [inline proto/binlog/pump.proto]: add  `start TS` and `commit TS`?  and describe how to use

Comment (WangXiangUSTC) [reply proto/binlog/pump.proto]: done, PTAL again

File: proto/binlog/pump.proto [hunk 2]
```diff
@@ -54,12 +54,26 @@ message Pos {
...
+    // The binlog's start ts, used in Prewrite, Commit and Rollback type binlog.
+    int64 startTs = 2;
+
+    // The binlog's commit ts, used only in Commit type binlog.
+    int64 commitTs = 3;
+}
+
+message Entity {
+    // The meta information of the binlog entity.
+    Meta meta = 1 [(gogoproto.nullable) = false];
```
Comment (GregoryIan) [inline proto/binlog/pump.proto]: I think it's better leave old definition，and append a new field

File: proto/binlog/pump.proto [hunk 3]
```diff
@@ -56,13 +56,25 @@ message Pos {
...
+// Meta saves the binlog's meta information.
+message Meta {
+    // The binlog's start ts, used in Prewrite, Commit and Rollback type binlog.
+    int64 startTs = 1;
+
+    // The binlog's commit ts, used only in Commit type binlog.
+    int64 commitTs = 2;
+}
+
 message Entity {
```
Comment (GregoryIan) [inline proto/binlog/pump.proto]: I mean
```
message Entity {
  Pos pos = 1 [(gogoproto.nullable) = false];
  bytes payload = 2;
  bytes checksum = 3;
  Meta meta = 4 [(gogoproto.nullable) = false];
}

Comment (WangXiangUSTC) [reply proto/binlog/pump.proto]: done, PTAL again

**Comment by GregoryIan** [comment]
LGTM

**Pull request merged**